benchmarks/src/jmh/java/com/linecorp/armeria/grpc/downstream/LargePayloadBenchmark.java

```diff
@@ -64,7 +63,7 @@
 
     @Setup
     public void setUp() {
-        server = new ServerBuilder()
+        server = Server.builder()
                 .serviceUnder("/", new GrpcServiceBuilder().addService(new BinaryProxyImplBase() {
```

Comment:
nit: indentation

Reply:
In this regard, 
(I asked about this issue to @trustin, He said I could decide whether I apply dot aligning or not(Since It is impossible or harms readability) as long as 
It's looked readable and neat that I 've aligned code follow my own rule.
I tried to aligning builder method change by dot( . )
Classes like this one are bound to exceed single line character limitation(112).
This code contains inner class in a lambda, I thought that indentation will be too deep.
So, I decided not to apply the aligning rule. I want to take your advice about this issues. 
My reservation is applying different aligning pattern by code would break consistency of code.
How should I handle this issue?

Reply:
Sorry about that I missed this comment.
As we discussed in the Slack, let's define a variable and use that to align with the `.`. :wink: